Cephfs mvn build doc command error out

**Alluxio Version:**
Alluxio master 2.6.0-SNAPSHOT-c6582b0cdcb83e41209cf00b809690588434a106

**Describe the bug**
Doc generation failed after PR https://github.com/Alluxio/alluxio/commit/3ef4728df196c17e4eb397f6612e41b68fe26791 merged. 
After removing the whole underfs/cephfs, the `mvn javadoc:aggregate` command passed.

**To Reproduce**
In linux environment, git clone https://github.com/alluxio/alluxio
git init
git fetch origin
git checkout origin/master
mvn clean install --no-transfer-progress -T 2C -DskipTests -Dmaven.javadoc.skip -Dcheckstyle.skip -Dlicense.skip -Dfindbugs.skip
mvn javadoc:aggregate --no-transfer-progress -T 2C -Dcheckstyle.skip -Dlicense.skip -Dfindbugs.skip

the javadoc:aggregate command error out
```
[INFO] Forking Alluxio Shaded Libraries - cephfs-hadoop 2.6.0-SNAPSHOT
[WARNING] Invalid POM for com.ceph.fs:cephfs-hadoop:jar:0.80.6, transitive dependencies (if any) will not be available, enable debug logging for more details
[INFO] Forking Alluxio Under File System - CephFS 2.6.0-SNAPSHOT
02:24:43  [WARNING] The POM for com.ceph:libcephfs:jar:0.80.5 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
[INFO] Forking Alluxio UI 2.6.0-SNAPSHOT
[WARNING] Invalid POM for com.ceph.fs:cephfs-hadoop:jar:0.80.6, transitive dependencies (if any) will not be available, enable debug logging for more details

02:27:14 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9:aggregate (default-cli) on project alluxio-parent: Execution default-cli of goal org.apache.maven.plugins:maven-javadoc-plugin:2.9:aggregate failed: For artifact {com.ceph:libcephfs:null:jar}: The version cannot be empty. -> [Help 1]
```

**Expected behavior**
Expect the cephfs not affecting the docs publish mvn build.

**Urgency**
High, it blocks the alluxio doc generation

**Additional context**
Add any other context about the problem here.

--- a/pom.xml
+++ b/pom.xml
@@ -165,8 +165,8 @@
     <update.check.host>https://diagnostics.alluxio.io</update.check.host>
     <findbugs.skip>false</findbugs.skip>
     <create.dependency.reduced.pom>false</create.dependency.reduced.pom>
-    <cephfs-hadoop.version>0.80.6</cephfs-hadoop.version>
-    <libcephfs.version>0.80.5</libcephfs.version>
+    <cephfs-hadoop.version>0.0.1</cephfs-hadoop.version>
+    <libcephfs.version>0.0.1</libcephfs.version>
   </properties>
 
   <modules>
@@ -215,12 +215,12 @@
         <version>1.78</version>
       </dependency>
       <dependency>
-        <groupId>com.ceph.fs</groupId>
+        <groupId>io.github.opendataio</groupId>
         <artifactId>cephfs-hadoop</artifactId>
         <version>${cephfs-hadoop.version}</version>
       </dependency>
       <dependency>
-        <groupId>com.ceph</groupId>
+        <groupId>io.github.opendataio</groupId>
         <artifactId>libcephfs</artifactId>
         <version>${libcephfs.version}</version>
       </dependency>

--- a/underfs/cephfs-hadoop/pom.xml
+++ b/underfs/cephfs-hadoop/pom.xml
@@ -30,12 +30,8 @@
   <dependencies>
     <!-- External dependencies -->
     <dependency>
-      <groupId>com.ceph.fs</groupId>
+      <groupId>io.github.opendataio</groupId>
       <artifactId>cephfs-hadoop</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.ceph</groupId>
-      <artifactId>libcephfs</artifactId>
     </dependency>
     <!-- Internal dependencies -->
     <dependency>

--- a/underfs/cephfs/pom.xml
+++ b/underfs/cephfs/pom.xml
@@ -28,7 +28,7 @@
   <dependencies>
     <!-- External dependencies -->
     <dependency>
-      <groupId>com.ceph</groupId>
+      <groupId>io.github.opendataio</groupId>
       <artifactId>libcephfs</artifactId>
     </dependency>
 

